commit 3bbd15676dfc077d7836e9d51810c1d6731f5789
Author: Palmer Cox <p@lmercox.com>
Date:   Sun Feb 23 16:41:18 2025 -0500

    Fix copy/paste error in FindPostgres.cmake
    
    In f51c6b1513e312002c108fe87d26e33c48671406, EXEC_PROGRAM was changed to
    execute_process. As part of that, it looks like the second and third
    invocations were accidentally changed.

diff --git a/saga-gis/cmake/modules/FindPostgres.cmake b/saga-gis/cmake/modules/FindPostgres.cmake
index f22806fd9..a4b6ec9ac 100644
--- a/cmake/modules/FindPostgres.cmake
+++ b/cmake/modules/FindPostgres.cmake
@@ -77,13 +77,13 @@ ELSE(WIN32)
       SET(POSTGRES_INCLUDE_DIR ${PG_TMP} CACHE STRING INTERNAL)
 
       # set LIBRARY_DIR
-      execute_process(COMMAND ${POSTGRES_CONFIG} --includedir
+      execute_process(COMMAND ${POSTGRES_CONFIG} --libdir
         OUTPUT_VARIABLE PG_TMP
         OUTPUT_STRIP_TRAILING_WHITESPACE)
       IF (APPLE)
         SET(POSTGRES_LIBRARY ${PG_TMP}/libpq.dylib CACHE STRING INTERNAL)
       ELSEIF (CYGWIN)
-        execute_process(COMMAND ${POSTGRES_CONFIG} --includedir
+        execute_process(COMMAND ${POSTGRES_CONFIG} --libs
           OUTPUT_VARIABLE PG_TMP
           OUTPUT_STRIP_TRAILING_WHITESPACE)
 
